
// ---: try And catch in JS - 2 :---

console.log("hello");
console.log("hello");

try
{
    console.log(a);
}catch(err){
    console.log("Caught an error... a is not defined");
    console.log(err);
}

console.log("hello2");
console.log("hello2");
console.log("hello2");
console.log("hello2");